packages/documentation/copy/en/reference/Containerized Execution.md

```diff
@@ -30,10 +30,10 @@ How to use this depends on whether your application is federated. You will need
 
 ### Using docker build and docker run
 
-Suppose your LF source file is `Foo.lf`. When you run `lfc` or use the IDE to generate code, a file called `Foo.Dockerfile` will appear in the `src_gen` directory. You can use this file to build a Docker image as follows. First, make sure you are in the same directory as the source file. Then issue the command:
+Suppose your LF source file is `Foo.lf`. When you run `lfc` or use the IDE to generate code, a file called `Foo.Dockerfile` will appear in the `src_gen/Foo` directory, see [Structure of an LF project](/docs/handbook/a-first-reactor#structure-of-an-lf-project) for more info. You can use this Dockerfile to build a Docker image as follows. First, navigate into the directory where `Foo.Dockerfile` is located. Then issue the command:
```

Comment:
@erlingrj, I realize that this PR was created a long time ago. I think it's still relevant, but outdated, because I think the file we produce now is just called `Dockerfile`. Could you please again verify the correctness of the proposed text? Thanks!

Reply:
I just checked and this PR is still correct

Reply:
Thanks!

Reply:
```suggestion
Suppose your LF source file is `src/Foo.lf`. When you run `lfc` or use the IDE to generate code, a file called `Dockerfile` and `docker-compose.yml` will appear in the `src_gen/Foo` directory, see [Structure of an LF project](/docs/handbook/a-first-reactor#structure-of-an-lf-project) for more info. You can now build Docker image as follows. First, navigate into the directory where `Dockerfile` is located. Then issue the command:
```